app/services/state_file/state_information_service.rb

```diff
@@ -1,16 +1,37 @@
 module StateFile
   class StateInformationService
     class << self
+      [
+        :download_form_name,
+        :intake_class,
+        :mail_voucher_address,
+        :pay_mail_online_link,
+        :refund_url,
+        :state_name,
+        :survey_link,
+        :tax_payment_url,
+        :vita_link,
+        :voucher_path,
+      ].each do |attribute|
+        define_method(attribute) do |state_code|
+          raise StandardError, "No state code '#{state_code}'" if !active_state_codes.include?(state_code)
+
+          value = STATES_INFO[state_code.to_sym][attribute]
+          raise StandardError, "State '#{state_code}' does not have '#{attribute}'" if !value
```

Comment:
this is me kind of breaking my own rule, the rule being "don't add code that isn't used just because it seems like a nice idea". there aren't currently any attributes in this list that are missing from a state so this line should never be called. now that i've said this out loud maybe i will delete it...